cachito/workers/tasks/utils.py

```diff
@@ -124,15 +124,15 @@ def task_with_state_check(*args, **kwargs):
             raise ValueError(
                 f"Failed during state check: no request_id found for {task_fn.__name__} task",
             )
-            return None
+            return
         request_state = get_request_state(request_id)
         if request_state != "in_progress":
             log.error(
                 "Skipping %s task because the request was not in_progress (was: %s)",
                 task_fn.__name__,
                 request_state,
             )
-            return None
+            return
```

Comment:
Sorry, I was not specific enough, What I meant is that `return` after `raise` is dead code, e.g.
```python
def f():
    raise ValueError
    return None  # unreachable statement
```
`return None` by itself (when the return statement is reachable) is completely fine.

Reply:
Haha sorry, my bad.